src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -242,18 +239,14 @@ public void stop() {
   public void open(final Collection<TopicPartition> partitions) {
     long startTime = System.currentTimeMillis();
     LOGGER.info(
-        Logging.logMessage(
-            "SnowflakeSinkTask[ID:{}]:open, TopicPartition number: {}",
-            this.id,
-            partitions.size()));
+        "SnowflakeSinkTask[ID:{}]:open, TopicPartition number: {}", this.id, partitions.size());
```

Comment:
can you add which partitions are assigned to this instance of the task. Requires a some computation but would worth in debugging. 
Also lets correct the message 
```suggestion
        "SnowflakeSinkTask[ID:{}]:open, TopicPartition size: {}", this.id, partitions.size());
```

Reply:
Can you add this in the below forEach loop too?
```suggestion
partitions.forEach(topicPartition -> Logging.logMessage("SnowflakeSinkTask[ID:{}] Assigned Partition:", topicPartition));
```

Reply:
This is out of scope for this pr i believe, I'll add this in a follow up one for the debugging scenario. created jira tracking this: https://snowflakecomputing.atlassian.net/browse/SNOW-665898

Reply:
Umm, okay. I think its fine to just add it here since it is only one more line but I will leave it up to you!